packages/algob/src/internal/deployer.ts

```diff
@@ -153,6 +153,13 @@ class DeployerBasicMode {
     return lsigAccount;
   }
 
+  /**
+   * Loads a logic signature account from checkpoint
+  */
```

Comment:
please add @param in description as well

Reply:
ok